node/.npmignore

```diff
@@ -0,0 +1,6 @@
+.eslintignore
+.eslintrc.json
+.prettierrc.js
+src/
```

Comment:
No need to include src/ since the package.json points to `dist/index.js` and `dist/index.d.ts`